src/js/relayclient/ServerHelper.js

```diff
@@ -172,20 +218,22 @@ class ServerHelper {
             let event = addedAndRemovedEvents[index]
             if (event.event === "RelayAdded") {
                 let args = event.returnValues
-                activeRelays[args.relay] = {
+                let relay = {
                     address: args.relay,
                     relayUrl: args.url,
                     transactionFee: args.transactionFee,
                     stake: args.stake,
                     unstakeDelay: args.unstakeDelay
                 }
+                relay.score = this.calculateRelayScore(relay) + this.addScoreRandomness()
```

Comment:
Didn't you move randomness to the "compareRelayScores" ?

Reply:
oops...